Makefile

```diff
@@ -47,6 +48,60 @@ help:
 # pip install command. Default: $(PIP_INSTALL)
 PIP_INSTALL = $(PIP) install
 
+deps-cuda: CONDA_EXE ?= /usr/local/bin/conda
+deps-cuda: export CONDA_PREFIX ?= /conda
+deps-cuda: PYTHON_PREFIX != $(PYTHON) -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])'
+deps-cuda:
+	curl -Ls https://micro.mamba.pm/api/micromamba/linux-64/latest | tar -xvj bin/micromamba
```

Comment:
These lines cause build warnings in the latest code (where line numbers changed):
```
Makefile:63: warning: overriding commands for target `deps-cuda'
Makefile:61: warning: ignoring old commands for target `deps-cuda'
```